docs/content/mobile/ios/setup.md

```diff
@@ -0,0 +1,13 @@
+# Setting up iOS support for the GDK
+
+## Get the dependencies for developing iOS games
+  1. Follow the steps in [Get the dependencies]({{urlRoot}}/setup-and-installing) and additionally install **iOS build support** for Unity.
+  2. Install iOS prerequisites
```

Comment:
Nit: use 1. for consistency.

Reply:
god, I thought I fixed it everywhere

Reply:
Sorry, it is my fault for introducing that in the first place :)